certbot-apache/tests/configurator_test.py

```diff
@@ -6,25 +6,22 @@
 import tempfile
 import unittest
 
-import mock
 # six is used in mock.patch()
```

Comment:
this comment is about `six` and should be moved down with it

Reply:
Done.